app/src/main/java/org/mozilla/fenix/tabstray/inactivetabs/InactiveTabs.kt

```diff
@@ -0,0 +1,294 @@
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import mozilla.components.browser.state.state.ContentState
+import mozilla.components.browser.state.state.TabSessionState
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.PrimaryText
+import org.mozilla.fenix.compose.SecondaryText
+import org.mozilla.fenix.compose.list.ExpandableListHeader
+import org.mozilla.fenix.compose.list.FaviconListItem
+import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.fenix.theme.Theme
+
+private val ROUNDED_CORNER_SHAPE = RoundedCornerShape(8.dp)
+
+/**
+ * Top-level Composable for displaying an expandable section of Inactive Tabs
+ *
+ * @param inactiveTabs List of [InactiveTabListItem] to display.
+ * @param expanded Boolean to determine whether to show the inactive tabs section expanded or collapsed.
+ * @param showAutoCloseDialog Boolean to determine whether to show the auto close inactive tabs dialog.
+ * @param onHeaderClick Called when the user clicks on the inactive tabs section header.
+ * @param onDeleteAllButtonClick Called when the user clicks on the delete all inactive tabs button.
+ * @param onAutoCloseDismissClick Called when the user clicks on the auto close dialog's dismiss button.
+ * @param onEnableAutoCloseClick Called when the user clicks on the auto close dialog's enable button.
+ * @param onTabClick Called when the user clicks on a specific inactive tab.
+ * @param onTabCloseClick Called when the user clicks on a specific inactive tab's close button.
+ */
+@Composable
+@Suppress("LongParameterList")
+fun InactiveTabsList(
+    inactiveTabs: List<InactiveTabListItem>,
+    expanded: Boolean,
+    showAutoCloseDialog: Boolean,
+    onHeaderClick: () -> Unit,
+    onDeleteAllButtonClick: () -> Unit,
+    onAutoCloseDismissClick: () -> Unit,
+    onEnableAutoCloseClick: () -> Unit,
+    onTabClick: (InactiveTabListItem) -> Unit,
+    onTabCloseClick: (InactiveTabListItem) -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp, vertical = 12.dp)
+            .clip(ROUNDED_CORNER_SHAPE)
+            .background(FirefoxTheme.colors.layer2)
+            .border(
+                width = 2.dp,
+                color = FirefoxTheme.colors.borderPrimary,
+                shape = ROUNDED_CORNER_SHAPE,
+            )
+            .padding(vertical = 8.dp)
+    ) {
+        InactiveTabsHeader(
+            expanded = expanded,
+            onClick = onHeaderClick,
+            onDeleteAllClick = onDeleteAllButtonClick,
+        )
+
+        if (expanded) {
+            if (showAutoCloseDialog) {
+                InactiveTabsAutoClosePrompt(
+                    onDismissClick = onAutoCloseDismissClick,
+                    onEnableAutoCloseClick = onEnableAutoCloseClick,
+                )
+            }
+
+            Column {
+                inactiveTabs.forEach { tab ->
+                    FaviconListItem(
```

Comment:
Not currently sure what to make of this yet. The favicon seems a bit small when I compare it to History, but I am not quite sure if this is because it's not loading the browser icon and just using the placeholder that just displays a letter for the URL.  Might want to throw some screenshots for UX to approve of or double check the sizing between a couple of screens.